robokassa/migrations/0003_load_source_type.py

```diff
@@ -7,16 +7,10 @@
 class Migration(DataMigration):
 
     def forwards(self, orm):
-        "Write your forwards methods here."
-        # Note: Remember to use orm['appname.ModelName'] rather than "from appname.models..."
-        from django.core.management import call_command
-        from os.path import dirname, join
-        fixture_path = join(dirname(__file__), "payment.sourcetype.json")
-        print "Loading fixture from %s" % fixture_path
-        call_command("loaddata", fixture_path)
```

Comment:
What do you dislike so much?

Reply:
This code depends on the actual code of the model. The model code was changed in our project. Because of this, it is not possible to properly run migrations.

Reply:
https://github.com/100maketov/django-oscar-robokassa/pull/10/commits/c5049b9bf465aee93d4c87b9cd62608d338ede7f